tests/endtoend/http_functions/common_libs_functions/common_libs_functions_stein/function_app.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import logging
+import azure.functions as func
+import numpy as np
+import requests
+import cv2
+from pandas import DataFrame
+from sklearn.datasets import load_iris
+import tensorflow as tf
+import plotly
+import keras
+
+
+app = func.FunctionApp()
+
+
+@app.route(route="keras_func")
+def keras_func(req: func.HttpRequest) -> func.HttpResponse:
+    logging.info('Python HTTP trigger function processed a request.')
+
+    res = "keras version: {}".format(keras.__version__)
+
+    return func.HttpResponse(res)
+
+
+@app.route(route="numpy_func")
+def numpy_func(req: func.HttpRequest) -> func.HttpResponse:
+    logging.info('Python HTTP trigger function processed a request.')
+
+    res = "array: {}".format(np.array([1, 2, 3], dtype=complex))
+
+    return func.HttpResponse(res)
+
+
+@app.route(route="opencv_func")
+def opencv_func(req: func.HttpRequest) -> func.HttpResponse:
+    logging.info('Python HTTP trigger function processed a request.')
+
+    img = cv2.imread("test.png", cv2.IMREAD_COLOR)
```

Comment:
You don't need to tests the library. To keep it simple just return and assert any attribute of opencv

Reply:
updated. Thanks